### DISCUSSION TITLE
v1 doesnt support the asynchronization？

### DISCUSSION THREAD

**Discussion by zjrwtx** [acheong08/ChatGPT#765] [General]
v1 doesnt support the asynchronization？

**Comment by acheong08** [comment]
It can but it won't because of rate limiting by OpenAI.

**Reply by acheong08** [reply]
It serves no purpose so i'm keeping it sync and sane

**Reply by zjrwtx** [reply]
all right，but it still needs the vpn to use in china，More troublesome

**Reply by acheong08** [reply]
Everything needs VPN in China. I just keep mine up 24/7

**Reply by zjrwtx** [reply]
got it，thanks
